packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -452,6 +454,7 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
 
   @override
   void didChangeDependencies() {
+    _mediaQueryData = MediaQuery.maybeOf(context);
```

Comment:
Actually, there's no guarantee that the GestureDetectors will be updated with the new gestureSettings when didChangeDependencies fires because `setCanDrag` may not execute?

Reply:
This is true - I don't think there is any reason for gesture settings to change frequently, but if the gesture detectors were already created we'd need to update them. That said, scrollable doesn't have access to the recognizer instances - and the recognizer instances don't have enough context to know generally if they should use the gesture settings

Reply:
The same is also true for the way is currently handles changes to physics....

I guess under the assumption that these rarely change, this is fine?

Reply:
Seems likely. I don't think there is a usecase for changing gesture settings besides at startup anyway